components/servo/webview_delegate.rs

```diff
@@ -586,6 +588,15 @@ pub trait WebViewDelegate {
 
     /// Request to display a notification.
     fn show_notification(&self, _webview: WebView, _notification: Notification) {}
+
+    ///
+    fn message_port_onmessage(
+        &self,
+        _webview: WebView,
+        _message_port: Rc<MessagePort>,
+        _data: JSValue,
+    ) {
+    }
```

Comment:
I suspect that this should be either a callback on the `MessagePort` itself or a delegate that's attached to the `MessagePort`.